BooruSharp.UnitTests/BooruTests.cs

```diff
@@ -375,9 +375,16 @@ public async Task SetHttpClientAsync()
         public async Task GetRandom2TagsAsync(Type t, string tag = "hibiki_(kantai_collection)", string tag2 = "school_swimsuit")
         {
             var booru = await Boorus.GetAsync(t);
-            var result = await booru.GetRandomPostAsync(tag, tag2);
-            Assert.Contains(result.Tags, t => t.Contains(tag));
-            Assert.Contains(result.Tags, t => t.Contains(tag2));
+            if (t.IsAssignableFrom(typeof(DanbooruDonmai)))
```

Comment:
It probably makes more sense to use this property to check whether the tags API limits the number of tags you can search with (and make it `public`). This will make tests more resilient to changes.
https://github.com/Xwilarg/BooruSharp/blob/ad21d80dba6aac58733fd1b694eb71b38179e5f3/BooruSharp/Booru/ABooru.cs#L135